srcpkgs/kColorPicker/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'kColorPicker'
+pkgname=kColorPicker
+version=0.0.1
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTS=ON -DBUILD_EXAMPLE=OFF -DBUILD_SHARED_LIBS=ON"
```

Comment:
Isn't building tests the default?

Reply:
It is explicitly disabled in the `CMakeLists.txt`